app/views/content/how-to-apply-for-teacher-training/teacher-training-interview.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Prepare for your teacher training interview
+title: Prepare for a teacher training interview
```

Comment:
This is longer than the recommended title length for SEO once the GIT/GOV.UK is added in (although I appreciate you are making it shorter here!)

Do we need to say `Prepare for a` here and in the h1/nav title - it's a bit inconsistent with the other pages in the How to apply section? Should it just be `Teacher training interviews`? But happy to go with your lead